src/lib/common/limits.h

```diff
@@ -94,6 +95,7 @@
 */
 #define  LOCAL_IP_V6  "::"
 #define  LOCAL_IP_V4  "0.0.0.0"
+#define  MAX_LEN_IP   64
```

Comment:
Actually I only moved this line from another place.
64 should be enough for IPv6. For IPv4 you only need 3+1+3+1+3+1+3+1 == 16 (terminating zero included)